src/components/Services/OtherServices/thriving-through-connection.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import ServicesSection from "../services-section";
+
+export default function ThrivingThroughConnectionSection() {
+  const body = (
+    <>
+      <p>
```

Comment:
The "Thriving Through Connection Supporting Young People Aged 11 - 17 years" text should be center aligned and have a margin so that there's space between the text and the border so it looks like this: 
![image](https://github.com/codersforcauses/starick/assets/22740095/ad19cb60-b5d9-4280-b3b5-c0d193cb7414)